src/app/debug/table/table.component.ts

```diff
@@ -511,13 +511,16 @@ export class TableComponent implements OnInit, OnDestroy {
   }
 
   downloadReports(exportBinary: boolean, exportXML: boolean): void {
-    const queryString: string = this.tableSettings.reportMetadata
-      .filter((report) => report.checked)
-      .reduce((totalQuery: string, selectedReport: any) => totalQuery + 'id=' + selectedReport.storageId + '&', '');
-    if (queryString === '') {
-      this.toastService.showWarning('No reports selected to download');
-    } else {
+    const selectedReports = this.tableSettings.reportMetadata.filter((report) => report.checked);
+
+    if (selectedReports.length > 0) {
+      let queryString: string = '';
+      selectedReports.forEach((report) => {
+        queryString += `id=${report.storageId}&`;
+      });
```

Comment:
Please use for...of instead of forEach()
```suggestion
      for (let report of selectedReport) {
        queryString += `id=${report.storageId}&`;
      }
```